release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -0,0 +1,860 @@
+:_content-type: ASSEMBLY
+[id="ocp-4-11-release-notes"]
+= {product-title} {product-version} release notes
+include::_attributes/common-attributes.adoc[]
+:context: release-notes
+
+toc::[]
+
+Red Hat {product-title} provides developers and IT organizations with a hybrid cloud application platform for deploying both new and existing applications on secure, scalable resources with minimal configuration and management overhead. {product-title} supports a wide selection of programming languages and frameworks, such as Java, JavaScript, Python, Ruby, and PHP.
+
+Built on {op-system-base-full} and Kubernetes, {product-title} provides a more secure and scalable multitenant operating system for today's enterprise-class applications, while delivering integrated application runtimes and libraries. {product-title} enables organizations to meet security, privacy, compliance, and governance requirements.
+
+[id="ocp-4-11-about-this-release"]
+== About this release
+
+{product-title} (link:https://access.redhat.com/errata/RHSA-2022:0056[RHSA-2022:0056]) is now available. This release uses link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.23.md[Kubernetes 1.23] with CRI-O runtime. New features, changes, and known issues that pertain to {product-title} {product-version} are included in this topic.
+
+Red Hat did not publicly release {product-title} 4.10.0 as the GA version and, instead, is releasing {product-title} 4.10.3 as the GA version.
```

Comment:
I suggest putting these in bold so that you remember to change them later.

Reply:
You can also mark out "Red Hat did not publicly release {product-title} 4.10.0 as the GA version and, instead, is releasing {product-title} 4.10.3 as the GA version." with `//` because you may or may not use it.